vxfreeswitch/voice.py

```diff
@@ -118,13 +120,32 @@ def stream_text_as_speech(self, message):
     def get_address(self):
         return self.uniquecallid
 
-    def output_message(self, text):
-        self.log("Sending text as speech: %r" % (text,))
-        return self.stream_text_as_speech(text)
-
-    def output_stream(self, url):
-        self.log("Playing back URL: %r" % (url,))
-        return self.playback(url)
+    def output_message(self, text, settings={}):
+        return self.stream_text_as_speech(text, settings=settings)
```

Comment:
Ah, I see it's replaced by log in `output_stream`. Please ignore previous comment.